document-css/doc.css

```diff
@@ -0,0 +1,57 @@
+@import "../assets/common-css/sidebar.css";
+@import "../assets/common-css/navbar.css";
+@import "../assets/common-css/containers.css";
+
+:root{
+    --primary-color : #1D3461;
+    --secondary-color : #829CBC;
+    --alert-color: rgb(241, 125, 16); 
+    --warning-color: rgb(224, 49, 18); 
+    --good-color: rgb(21, 150, 21); 
+    --font-color: #201b1b ; 
+    --background-color: #eeeef7 ; 
+    --whitesmoke-color : whitesmoke;
+    --font-color: #201b1b ; 
+    --background-color: #eeeef7 ; 
+    --eg-container-bg : #dbe0e7;
+
+    --head-type-font : 1.7rem;
+    --type-margin : 1.875rem;
+    --content-padding : 1.25rem;
+    --content-font : 1.1875rem;
+    --container-gap : 1.875rem;
+    --container-margin-top: 1.25rem;
+    --doc-container-margin-top: 5.625rem;
+    --doc-container-gap : 3.125rem; 
+    --border-radius: 5px;
+    --container-flex-basis : 25rem;
+}
+html{
+    font-family: 'Montserrat', sans-serif ;
+    background-color: whitesmoke;
+}
+.document-container{
+    display: flex;
+    margin-top: 90px;
+    gap: 3.125rem;
+}
+.intro-head, .install-head{
+    color: var(--font-color);
+    font-size: 1.7rem;
+
+}
+.intro-head{
+    margin: -5px;
+}
+.install-head{
+    margin-top: 30px;
```

Comment:
for unit use` rem` as you are using it everywhere

Reply:
Yep, changing it right away.